.scripts/test_all.sh

```diff
@@ -26,6 +26,30 @@ else
   go get -u github.com/hashicorp/consul/...
 fi
 
+install_etcd_pkgs() {
+  go get -u github.com/coreos/etcd
+  go get -u github.com/gogo/protobuf/gogoproto
+  go get -u google.golang.org/grpc
+  go get -u github.com/coreos/go-semver
+  go get -u github.com/ugorji/go/codec
```

Comment:
Is there a reason why you are pulling in the sub-dependencies manually? Is it not enough to do `go get github.com/coreos/etcd` and `go get github.com/mwitkow/go-etcd-harness`?

Reply:
`go get github.com/coreos/etcd` pulls the latest unstable (and does not compile from my prior attempts). I could try the `gopkg.in` trick you suggested and see if it helps.

Reply:
> (and does not compile from my prior attempts)

That's unexpected, it works fine for me. What error are you seeing? etcd even vendors its dependencies so we should not need to install etcd's dependencies at all.

Reply:
hm I can try it again on a blank ec2 box and see whether I'm getting errors / things are just broken on my local environment somehow

Reply:
Using go.etcd.io/etcd/clientv3 now